Artisan/CraftingLogic/ExpertSolver/Simulator.cs

```diff
@@ -0,0 +1,307 @@
+﻿using Artisan.CraftingLogic.CraftData;
+using Artisan.RawInformation.Character;
+using System;
+
+namespace Artisan.CraftingLogic.ExpertSolver;
+
+public static class Simulator
```

Comment:
This is a essentially a copy-paste of what I had in my experimental plugin, not sure what to do with it, it's not really specific to expert crafts, and I think there's some duplication with existing code.